rmgpy/molecule/draw.py

```diff
@@ -176,7 +176,10 @@ def draw(self, molecule, file_format, target=None):
         surface_sites = []
         for atom in self.molecule.atoms:
             if atom.is_hydrogen() and atom.label == '':
-                atoms_to_remove.append(atom)
+                for bond in atom.bonds.values():
```

Comment:
I was going to say I prefer my loop because it can break on finding the first H bond instead of having to evaluate all the bonds and then check the list, which your list comprehension does. Then I realized you can pass a _generator_ to the `any()` method instead of a list, and then it can abort the loop on the first hit! Just remove the `[]` from your code!

Any time you see `any([...])` or `all([...])` you should be very suspicious that the square brackets are necessary. You can almost always remove them and get code that runs faster and uses less memory.
https://eklitzke.org/generator-comprehensions-and-using-any-and-all-in-python